### PR TITLE
Improve WASM app

### PR DESCRIPTION
**Pull request by Meakk**
- Fix build guide with Windows
- Do not try to cache baked HDRI when running on WASM
- Add SSAO, HDRI ambient to rendering options
- Add dark/light mode switch

### CODE REVIEW

File: doc/dev/BUILD_WASM.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ F3D can be built in WebAssembly using emscripten in order to embed it into a web
 It is still experimental and only a small fraction of the libf3d public API is bound.
 An example can be seen at https://f3d.app/web
 
-This guide is describing how to build VTK and F3D with emscripten using Docker on Linux.
+This guide is describing how to build VTK and F3D with emscripten using Docker on Linux or Windows (WSL).
```
Comment (mwestphal) [inline doc/dev/BUILD_WASM.md:7]: WSL is not Windows, not sure its worth mentionning.

**Review by mwestphal — CHANGES_REQUESTED** [review]
small discussion

File: doc/dev/BUILD_WASM.md (hunk 1 above)
Comment (Meakk) [reply doc/dev/BUILD_WASM.md:7]: It's using the WSL layer silently through Docker. I think it's worth mentioning this guide is valid to run on Windows.

Comment (Meakk) [reply doc/dev/BUILD_WASM.md:7]: I can remove ` (WSL)` if you think it's better tho

Comment (mwestphal) [reply doc/dev/BUILD_WASM.md]: Do the command need to be run in a WSL terminal or does it work in any terminal assuming docker is installed and in the PATH ?

Comment (Meakk) [reply doc/dev/BUILD_WASM.md]: Any terminal. If WSL is not installed, Docker will refuse to run the image.

**Review by mwestphal — APPROVED** [review]

**Pull request merged**